src/Settings.h

```diff
@@ -1,10 +1,271 @@
 #ifndef SETTINGS_H
 #define SETTINGS_H
 
+#include <iostream>
+#include <string>
 #include <unordered_map>
+#include <utility>
+#include <vector>
+#include <QFontDatabase>
+#include <QLocale>
 #include <QSettings>
 #include <QVariant>
 
+// ref: https://www.techiedelight.com/use-std-pair-key-std-unordered_map-cpp/
+typedef std::pair<std::string, std::string> SectionKeyPair;
+
+struct pair_hash
+{
+    template<class T1, class T2>
+    std::size_t operator()(const std::pair<T1, T2>& pair) const
+    {
+        return std::hash<T1>()(pair.first) ^ std::hash<T2>()(pair.second);
+    }
+};
+
+struct pair_compare
+{
+    template<class T1, class T2>
+    bool operator()(const std::pair<T1, T2>& lhs, const std::pair<T1, T2>& rhs) const
+    {
+        if (lhs.first == rhs.first)
+            return lhs.second < rhs.second;
+        else
+            return lhs.first < rhs.first;
+    }
+};
+
+namespace szINI
```

Comment:
I don't get here the meaning of `sz`. And shouldn't be `SzIni`? In fact, CamelCase is many times a source of inconsistencies when acronyms are used. Due to historical reasons, we have a mixture of styles, but maybe we should start to follow something consistently.